public/index.css

```diff
@@ -1,5 +1,8 @@
 * {
-  margin: 0;
-  padding: 0;
```

Comment:
This * can have weird consequences for margin and padding, so let's just throw those on the body (where I think we want to fix it)